jlibra-core/src/main/java/dev/jlibra/client/LibraJsonRpcTransport.java

```diff
@@ -34,8 +38,22 @@ public String pass(String request) throws IOException {
         post.setEntity(new StringEntity(request, UTF_8));
         post.setHeader(HttpHeaders.CONTENT_TYPE, CONTENT_TYPE_JSON);
         post.setHeader(HttpHeaders.USER_AGENT, USER_AGENT);
+
+        log.debug("Request: {}", log.isDebugEnabled() ? EntityUtils.toString(post.getEntity()) : "");
```

Comment:
Is isDebugEnabled still necessary? I thought there are other possibilities like lazy evaluation with lambdas (sorry, don't know details, but this looks old-fashioned).

Reply:
I checked this, the lambda support will be in version 2 of slf4j

Reply:
Ah ok, that's what I am using otherwise obviously. But here it doesn't make sense, so the old solution is fine.

Reply:
I would add `if` statement around `log.debug` rather than inside.